tests/unit/test_isac_mapping.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from mappings.chicagoISAC import ChicagoISACMapping
+
+
+class TestISACMapping:
```

Comment:
FYI, with `pytest`, you don't really need classes here! You can just define all these functions in the module scope and the tests will just work. Feel free to merge as is, just an fyi!